src/GeneralStateTestsFiller/stBugs/randomStatetestDEFAULT-Tue_07_58_41-15153-575192Filler.json

```diff
@@ -0,0 +1,82 @@
+{
+  "randomStatetestDEFAULT-Tue_07_58_41-15153-575192": {
+    "_info": {
+      "comment": "This is a canon example of a test found by fuzzing with EVMlab, demoing how a suicide-created-but-empty account has a non-zero codehash in geth",
+      "filledwith": "evmlab",
+      "lllcversion": "not available",
+      "source": "not available",
+      "sourceHash": "not available"
+    },
+    "expect" : [
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : [">=Constantinople"],
+                "result" : {
+                    "0x1000000000000000000000000000000000000000" : {
+                            "balance" : "0x3562",
+                            "code" : "0x61dead6000600060006000600061dead5af162abcdef3f600155",
+                            "nonce" : "0x1c",
+                            "storage" : {
+                            }
+                        },
+                        "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                            "balance" : "0x5d8fdd3ff5426352",
+                            "code" : "",
+                            "nonce" : "0x1d",
+                            "storage" : {
+                            }
+                   }
+                }
+            }
+    ],
+    "env": {
+      "currentCoinbase": "0x1000000000000000000000000000000000000000",
+      "currentDifficulty": "0x20000",
+      "currentGasLimit": "0x1312D00",
+      "currentNumber": "1",
+      "currentTimestamp": "1000",
+      "previousHash": "5fa368f13facaec6bf82d19c5d075fd3386be8a4265ed2d90c29b312a09b6d9d"
+    },
+    "pre": {
+      "000000000000000000000000000000000000dead": {
+        "balance": "0x00",
+        "code": "0x62abcdefff",
+        "nonce": "0x1c",
+        "storage": {}
+      },
+      "1000000000000000000000000000000000000000": {
+        "balance": "0x00",
+        "code" : "0x61dead6000600060006000600061dead5af162abcdef3f600155",
```

Comment:
`PUSH2 0xDEAD PUSH1 0x0 PUSH1 0x0 PUSH1 0x0 PUSH1 0x0 PUSH1 0x0 PUSH2 0xDEAD GAS CALL PUSH3 0xABCDEF EXTCODEHASH PUSH1 0x1 SSTORE`

Reply:
Note that this is not calling `EXTCODEHASH` on an address which just selfdestructed, it is instead calling `EXTCODEHASH` on a contract which was the beneficiary of a `SELFDESTRUCT` call but received 0 wei, triggering account cleanup.